bin/feature_toggle_config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Generally: Can this live in a test folder?

Reply:
Sure moved this to test folder.